android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -100,7 +106,12 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
         cancel()
       } else {
         view.onTouchEvent(event)
-        if ((state == STATE_UNDETERMINED || state == STATE_BEGAN) && view.isPressed) {
+
+        // We have to explicitly check for ReactTextView, since its `isPressed` flag is not set to `true`,
+        // in contrast to Touchable
+        val shouldActivate = view.isPressed || view is ReactTextView
```

Comment:
We can make a hook method out of this

Reply:
Agree, done in [40e5322](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/40e5322e399725e1c741a49d75e3b3a402f43895)